src/index.html

```diff
@@ -11,7 +11,172 @@
+          name="birthDate"
+          required
+          >
+        </label>
+
+        <label class="input">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="term"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="input">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          required
+          placeholder="email@example.com"
+          multiple
+          >
+        </label>
+        <label class="input">
+          Password:
+          <input
+          type="password"
+          name="password"
+          required
+          minlength="5"
+          maxlength="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="catslover"
+            value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="catslover"
+            value="no"
+            >
+            No
+          </label>
+        </div>
+        <label class="input">
+          What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          >
+        </label>
+        <label class="input">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          step="1"
+          >
+        </label>
+        <label class="input">
+          What are your favorite brand of cars?
+          <select
+          name="car"
+          required
+          multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option
+            value="empty"
+            disabled
+            selected
+            >
+            </option>
+          </select>
+        </label>
+        <label class="input">
+          How do you rate our work?
+          <input
+          type="range"
+          name="rating"
+          min="0"
+          max="100"
+          value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Additional info:</legend>
+        <label class="input">
+          Comments:
+          <textarea name="feedback"></textarea>
+        </label>
+        <label class="input">
+          Would you recommend us?
+          <select name="rating">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button
+      type="submit"
+      value="Submit"
+      >
+      Submit
+      </button>
+
+    </form>
```

Comment:
```suggestion
      </button>
    </form>
```
Don't leave empty lines between parent and child elements